docs/cimpl6LanguageReference.md

```diff
@@ -388,7 +375,7 @@ Examples:
 
 ### Value
 
-`Value` represents the data type(s) an `Element` can accept. This keyword can only be used when defining an `Element`. Each `Element` must have exactly one `Value`, although the `Value` itself can be a choice of several data types. `Value` choices can be [Primitives](#primitives), [Element](#element), [Group](#group), or [Entry](#entry). A `Value` may be inherited and constrained in the child class.
+`Value` represents the data type(s) an `Element` can accept. This keyword can only be used when defining an `Element`. Each `Element` must have exactly one `Value`, although the `Value` itself can be a choice of several data types. A `Value` choice can be a [primitive](#primitives), [Element](#element), [Group](#group), or [Entry](#entry). A `Value` may be inherited and constrained in the child class.
```

Comment:
**an** [Element](#element), **a** [Group](#group), or **an** [Entry](#entry).   (only bolded so you can see the suggested change.

Reply:
Fixed.